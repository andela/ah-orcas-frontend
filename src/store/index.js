import { createStore, applyMiddleware } from 'redux';
import { createLogger } from 'redux-logger';
import thunk from 'redux-thunk';

import allReducers from '../reducers';

const logger = createLogger();
export const store = createStore(allReducers, applyMiddleware(logger, thunk));
export default store;
