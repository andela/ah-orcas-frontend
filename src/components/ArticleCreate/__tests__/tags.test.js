import React from 'react';
import renderer from 'react-test-renderer';
import Tag from '../tagComponent';
import store from '../../../store';

it('renders correctly', () => {
  const tree = renderer
    .create(<Tag store={store} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
