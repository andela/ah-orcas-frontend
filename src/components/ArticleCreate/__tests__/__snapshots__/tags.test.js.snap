// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "float": "left",
      "width": "100%",
    }
  }
>
  <div
    className="form-group"
  >
    <span
      htmlFor="tag"
    />
    <div
      className="row"
    >
      <input
        className="form-control description"
        name="title"
        onChange={[Function]}
        placeholder=""
        style={
          Object {
            "border": "1px solid",
            "marginLeft": "1.5%",
            "marginRight": "2%",
            "marginTop": "1%",
            "width": "47%",
          }
        }
        value=""
      />
      <input
        className="form-control description"
        name="tag"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="add a tag"
        style={
          Object {
            "border": "1px solid",
            "margin": "1%",
            "width": "47%",
          }
        }
        value=""
      />
    </div>
  </div>
</div>
`;
